metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -521,3 +511,94 @@ def _convert_value(param):
             "See the run in the UI at %s" % run_url,
             bold=True,
         )
+
+
+@argo_workflows.command(help="Delete the Argo workflow")
+@click.option(
+    "--authorize",
+    default=None,
+    type=str,
+    help="Authorize the deletion with a production token",
+)
+@click.pass_obj
+def delete(obj, authorize=None):
+    validate_token(obj.workflow_name, obj.token_prefix, obj, authorize, "delete")
+    obj.echo("Deleting workflow *{name}*...".format(name=obj.workflow_name))
+
+    # Always try to delete the schedule. Failure in deleting the schedule should not
+    # be treated as an error, due to any of the following reasons
+    # - there might not have been a schedule, or it was deleted by some other means
+    # - retaining these resources should have no consequences as long as the workflow deletion succeeds.
+    # - regarding cost and compute, the significant resources are part of the workflow teardown, not the schedule.
+    schedule_deleted = ArgoWorkflows.remove_schedule(obj.workflow_name)
+    if schedule_deleted:
+        obj.echo("   Removed schedule")
```

Comment:
Can we instead mention which cronworkflow was deleted here. Maybe just `Deleting cronworkflow foo as well...` without the tabbed indent